docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -4,36 +4,330 @@
+
+     $ # Check that AWS sees the shared-credentials-files
+     $ aws configure list
+           Name                    Value             Type    Location
+           ----                    -----             ----    --------
+        profile                <not set>             None    None
+     access_key     ****************xxxx shared-credentials-file
+     secret_key     ****************xxxx shared-credentials-file
+         region                <not set>             None    None
+
+     $ # Validate that credentials are working
+     $ sky check aws
+
+
+.. _several-aws-profiles:
+
+Using several profiles or accounts
+----------------------------------
+
+You can use different AWS profiles or accounts to launch different clusters. SkyPilot will remember the owner identity of each cluster and properly protects any "write" operations. All clusters are shown in ``sky status``.
+
+Example of mixing a default profile and an SSO profile:
+
+.. code-block:: console
+
+    $ # A cluster launched under the default AWS identity.
+    $ sky launch --cloud aws -c default
+
+    $ # A cluster launched under a different profile.
+    $ AWS_PROFILE=AdministratorAccess-12345 sky launch --cloud aws -c my-sso-cluster
+
+
+.. _aws-troubleshooting:
+
+Troubleshooting issues
+----------------------
+
+If your credentials are not being picked up, or you're seeing the wrong credentials in SkyPilot, here are some steps you can take to troubleshoot:
+
+1. **Check** ``aws configure list``. This command should show the currently configured credentials.
+
+   If you have static credentials set up correctly, you should see something like this:
+
+   .. code-block:: console
+
+       $ aws configure list
+             Name                    Value             Type    Location
+             ----                    -----             ----    --------
+          profile                <not set>             None    None
+       access_key     ****************xxxx shared-credentials-file
+       secret_key     ****************xxxx shared-credentials-file
+           region                <not set>             None    None
+
+   If you have SSO credentials set up correctly, you should see something like this:
+
+   .. code-block:: console
+
+       $ aws configure list
+             Name                    Value             Type    Location
+             ----                    -----             ----    --------
+          profile                <not set>             None    None
+       access_key     ****************xxxx              sso
+       secret_key     ****************xxxx              sso
+           region                <not set>             None    None
+
+2. **Check** ``sky check aws``. This should show whether SkyPilot is picking up the credentials and has the necessary permissions.
+
+   .. code-block:: console
+
+       $ sky check aws
+       Checking credentials to enable clouds for SkyPilot.
+         AWS: enabled [compute, storage]
+       ...
```

Comment:
How about using `sky check aws -v` as the first step? It will show what credential is being used and also see if things has been setup.